api/v1/lib/master/calls/calls.go

```diff
@@ -221,6 +228,32 @@ func DestroyVolumes(a mesos.AgentID, v ...mesos.Resource) *master.Call {
 	}
 }
 
+// GrowLocalVolume grows a persistent volume on an agent's ROOT or PATH disks. The request is forwarded asynchronously
+// to the Mesos agent where the persistent volume is located.
+func GrowLocalVolume(a mesos.AgentID, volume mesos.Resource, addition mesos.Resource) *master.Call {
```

Comment:
nit: it's more idiomatic to say `volume, addition mesos.Resource` (since they've of the same type)

Reply:
also: mesos.AgentID is optional in the protos, so it should be a pointer here